README.md

```diff
@@ -29,41 +29,63 @@ file for configuring OpenId Connect variables
 
 ```yaml
 itkdev_openid_connect:
-  cache_options: 
+  cache_options:
     cache_pool: 'cache.app' # Cache item pool for caching discovery document and CLI login tokens
-  openid_provider_options:
-    configuration_url: '%env(CONFIGURATION_URL)%' # url to OpenId Discovery document
-    client_id: '%env(CLIENT_ID)%' # Client id assigned by authorizer
-    client_secret: '%env(CLIENT_SECRET)%' # Client password assigned by authorizer
-    callback_uri: '%env(CALLBACK_URI)%' # Callback URI registered at identity provider
   cli_login_options:
     cli_redirect: '%env(CLI_REDIRECT)%' # Redirect route for CLI login
+  openid_providers:
+    # Define one or more providers
+    admin:
+      options:
+        metadata_url: '%env(ADMIN_OIDC_METADATA_URL)%'
+        client_id: '%env(ADMIN_OIDC_CLIENT_ID)%'
+        client_secret: '%env(ADMIN_OIDC_CLIENT_SECRET)%'
+        # Specify redirect URI
+        redirect_uri: '%env(ADMIN_OIDC_REDIRECT_URI)%'
+    user:
+      options:
+        metadata_url: '%env(ADMIN_OIDC_METADATA_URL)%'
+        client_id: '%env(ADMIN_OIDC_CLIENT_ID)%'
+        client_secret: '%env(ADMIN_OIDC_CLIENT_SECRET)%'
+        # Use route as redirect URI
+        redirect_route: 'default'
```

Comment:
Why is this different from the admin block? Should this be 
```redirect_uri: '%env(USER_OIDC_REDIRECT_URI)%'```

Reply:
`redirect_route` (and `redirect_route_parameters`) can be used as an alternative to `redirect_uri`. I'll update the example to make this more clear.